vcpkg/TOC.yml

```diff
@@ -232,6 +232,8 @@
         href: maintainers/functions/internal/z_vcpkg_restore_pkgconfig_path.md
       - name: z_vcpkg_setup_pkgconfig_path
         href: maintainers/functions/internal/z_vcpkg_setup_pkgconfig_path.md
+  - name: Supplying Usage

```

Comment:
Docs uses sentence casing, e.g. Supplying Usage.  This title provides no context. Would it make sense to title this "Usage file guidance"?